src/main/java/Storage.java

```diff
@@ -0,0 +1,135 @@
+     * is a directory rather than a regular file,
+     * does not exist but cannot be created, or
+     * cannot be opened for any other reason.
+     */
+    public void writeToFile(String filepath, String textToAdd) throws IOException {
+        FileWriter typer = new FileWriter(filepath);
+        typer.write(textToAdd);
+        typer.close();
+    }
+
+    /**
+     * Counts the number of tasks in the list.
+     *
+     * @param filename File that the tasks are in.
+     * @return Returns the number of tasks.
+     * @throws IOException If the named file exists but
+     * is a directory rather than a regular file,
+     * does not exist but cannot be created, or
+     * cannot be opened for any other reason.
+     */
+    public static int countLines(String filename) throws IOException {
+        try (InputStream inputs = new BufferedInputStream(new FileInputStream(filename))) {
+            byte[] characters = new byte[1024];
+            int readCharacters = inputs.read(characters);
+            if (readCharacters == -1) {
+                // no lines to read
+                return 0;
+            }
+            int count = 0;
+            while (readCharacters == 1024) {
+                for (int i = 0; i < 1024; ) {
+                    if (characters[i++] == '\n') {
+                        ++count;
+                    }
+                }
+                readCharacters = inputs.read(characters);
+            }
+            // count remaining characters
+            while (readCharacters != -1) {
+                for (int i = 0; i < readCharacters; ++i) {
+                    if (characters[i] == '\n') {
+                        ++count;
+                    }
+                }
+                readCharacters = inputs.read(characters);
+            }
+
+            return count == 0 ? 1 : count;
+        }
+    }
+
+    /**
+     * Loads the task into the tasklist
+     * in TaskList from the file.
+     *
+     * @return ArrayList that has been copied from the file.
+     * @throws IOException If the named file exists but is a directory rather than a regular file,
+     * does not exist but cannot be created, or cannot be opened for any other reason.
+     * @throws DukeException If there is nothing in the file to be loaded,
+     * this exception will be thrown.
+     */
+    public ArrayList<Task> load() throws IOException, DukeException {
+        File f = new File(file);
+        Scanner sc = new Scanner(f);
+        ArrayList<Task> tempList;
+        if (countLines(file) == 0) {
+            throw new DukeException("Woahsies wavy! There is nothing in this file!");
+        } else {
+            while (sc.hasNext()) {
+                String task = sc.nextLine();
+                int index = task.indexOf("[");
+                String taskType = task.substring(index + 1, index + 2);
+                int spaceIndex = task.indexOf(" ");
+                switch (taskType) {
```

Comment:
There's a specific convention to follow for switch statements! https://nus-cs2103-ay1920s1.github.io/website/coding-standards/java/intermediate.html